File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.js
// 백준 알파벳 찾기
const S = require('fs').readFileSync('/dev/stdin').toString().trim();
const alphabet = Array.from({length : 26},(v,i) => String.fromCharCode(i+97));
let result = [];
for(let i = 0 ; i < alphabet.length ; i++){
    //result += (` ${S.indexOf(alphabet[i])}`);
    result.push(S.indexOf(alphabet[i]));
}
console.log(result.join(' '));

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String S = br.readLine();
        String alphabet[] = new String[26];
        Arrays.fill(alphabet,"-1"); // -1로 초기화

        for (int i = 0 ; i < S.length() ; i++){
            int idx = S.charAt(i) - 97;
            char ch = S.charAt(i);
            alphabet[idx] = String.valueOf(S.indexOf(ch));
        }
        
        System.out.print(String.join(" ",alphabet));
    }
}
